example/src/main/tut/typeclass/Functor.md

```diff
@@ -5,10 +5,6 @@ title:  "Functor"
 
 # Functor
 
-The `Functor` type class represents a type that can be mapped over.
```

Comment:
I think this is a more useful line to keep than the one about "(mathematical) functor"s.

Reply:
I can agree with that :)